DATASETS.md

```diff
@@ -31,6 +31,75 @@ By default, downloading the data for train/val/example scenes also pulls in the
 
 By default the download script will only download what is needed for habitat-sim.  You can add `_full` to the uid to download the raw glbs and the obj+mtl's in addition to what is needed for use with habitat-sim.
 
+### Loading semantics for HM3D
+
+First, ensure that the corresponding file structure exists (see minival example below):
+
+```
+> ls <PATH TO HM3D>
+hm3d_annotated_basis.scene_dataset_config.json    train/
+hm3d_basis.scene_dataset_config.json              val/
+minival/
+
+> ls <PATH TO HM3D>/minival
+00800-TEEsavR23oF/    00806-tQ5s4ShP627/
+00801-HaxA7YrQdEC/    00807-rsggHU7g7dh/
+00802-wcojb4TFT35/    00808-y9hTuugGdiq/
+00803-k1cupFYWXJ6/    00809-Qpor2mEya8F/
+00804-BHXhpBwSMLh/    hm3d_annotated_minival_basis.scene_dataset_config.json
+00805-SUHsP6z2gcJ/    hm3d_minival_basis.scene_dataset_config.json
+
+> ls <PATH TO HM3D>/minival/00800-TEEsavR23oF/
+TEEsavR23oF.basis.glb       TEEsavR23oF.basis.navmesh
+TEEsavR23oF.semantic.glb    TEEsavR23oF.semantic.txt
+```
+Note that there may be more files in `<PATH TO HM3D>/minival/00800-TEEsavR23oF/` if the full HM3D dataset is downloaded. Most importantly, ensure that the `hm3d_annotated_*`, `*.semantic.glb`, and `*.semantic.txt` files are present.
+
+To load semantic annotations in habitat-sim:
+* Enable the semantic sensor
+* Set the `scene_dataset_config_file` configuration variable
+
+A simple example below:
```

Comment:
Noob question: Where can a user use this script?

Reply:
Same question here. Can we tell the reader where to do this?

Reply:
It shouldn't matter as long as <PATH TO HM3D> is set correctly :)